network/config_test.go

```diff
@@ -15,3 +15,19 @@ func TestConf(t *testing.T) {
 	conf.EnableRelay = false
 	assert.NoError(t, conf.SanityCheck())
 }
+
+func TestValidateAddress(t *testing.T) {
+	validAddresses := []string{
+		"/ip4/127.0.0.1/tcp/21777/p2p/QmUv7r8T5NT3xQ2Zzvbg53uSdVccQ6LbdY2wAju14HsWyc",
```

Comment:
For valid address you can use: "/ip4/127.0.0.1"
For invalid addresses you can use: "", "/ip4", "127.0.0.1"

Keep test simple